src/controllers/auth-controller.ts

```diff
@@ -24,51 +24,64 @@ export default class AuthController extends AbstractController {
     this.router.post('/refresh', this.refresh);
 
     /* use auth.required to secure route */
-    this.router.get('/test', auth.required, this.test);
+    this.router.get('/test', authRequired, this.test);
 
     return this.router;
   }
 
-  private signUp = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  private signUp = async (req: Request, res: Response, next: NextFunction) => {
     try {
+      const { email, password, firstName, lastName } = req.body;
+      if (!email || !password || !firstName || !lastName) {
```

Comment:
By sheme `firstName` & `lastName` aren't required fields

Reply:
I will apply validation middleware next time.

Reply:
But while you didnt do it -- please, remove these checks for last and first names)

Reply:
fixed